notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>Cell needs a link back to the documentation at the end of the cell:</p><p> </p><p>Learn more about [...](...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
What should this link back to?

Reply:
To a web doc that describes Bigframes